Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -777,7 +777,7 @@ public virtual void Give(Player player, Item item, bool displayMessage = true)
                 if (displayMessage)
                     ShowPickedUpMessage(player);
 
-                Timing.CallDelayed(0.05f, () => OnAcquired(player));
```

Comment:
frame delay necessary to ensure the client has actually processed receiving the item before you try to do something with it in the OnAcquired override method.

Reply:
i don't get why we need client to have it ?